__tests__/integration/schema-version.test.ts

```diff
@@ -1,15 +1,17 @@
-import { fql } from "../../src";
-import { getClient } from "../client";
+import { Client, fql } from "../../src";
+import { newDB } from "../client";
 
 describe("schema version is returned by the client", () => {
-  const client = getClient();
+  let client: Client;
+
+  // make a fresh db each run
+  beforeAll(async () => {
+    client = await newDB("SchemaTest");
+  });
+
   it("returns the schema version", async () => {
     const resTs = await client.query(fql`
-      if (Collection.byName("TestColl") == null) {
-        Collection.create({ name: "TestColl" })
-      } else {
-        TestColl.definition.update({})
-      }
```

Comment:
Did this not work because `.update({})` is effectively a no-op? What if we just incremented some meta-data? Would changing the `data` not change the schema version.

Reply:
I'm not sure if it would change the schema version. Its easier to have a clean DB each time the test is ran, so I just went ahead and wrote the helper.

If you'd prefer to mess around with `update` I can change the PR, I just thought that making a fresh DB would lead to more reliable and simpler tests.